### PR TITLE
OpenADR initiative

### PR DESCRIPTION
**Pull request by erikjee**

### CODE REVIEW

File: content/initiatives/automated-demand-response.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+blogposts = [
+]
++++
+
+### Automated Demand Response
+
+With the rise of renewable energy resources and ongoing electrification, automated demand response plays a crucial role in optimizing the use of our electricity grid. 
+
+This project is building a modern, open-source Rust implementation of OpenADR 3.0, the latest version of OpenADR, the automated demand response standard that's being developed by the [OpenADR Alliance](https://www.openadr.org/).
```
Comment (erikjee) [inline content/initiatives/automated-demand-response.md:20]:
```suggestion
This project is building a modern, open-source implementation of OpenADR 3.0, the latest version of OpenADR, the automated demand response standard that's being developed by the [OpenADR Alliance](https://www.openadr.org/).
```

File: content/initiatives/automated-demand-response.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+After inital funding by ElaadNL and Tweede golf in 2024, the project is currently (November 2204) **seeking a total of EUR 40.000 in funding** for:
+
+**In Dec 2024 / Jan 2025:**
++ Doing a first field-test-ready release 
+
+**During 2025:**
++ Ongoing development and maintenance
++ Continued community building and contributor support
++ Additional releases when new (minor) versions of the OpenADR starndard are published
```
Comment (erikjee) [inline content/initiatives/automated-demand-response.md:46]:
```suggestion
+ Additional releases when new (minor) versions of the OpenADR standard are published
```

File: templates/components/nav.html [hunk 3]
```diff
@@ -22,6 +22,7 @@
           <a class="subitem" href="/initiatives/time-synchronization">Time synchronization</a>
           <a class="subitem" href="/initiatives/privilege-boundary">Privilege boundary</a>
           <a class="subitem" href="/initiatives/data-compression">Data compression</a>
+          <a class="subitem" href="/initiatives/automated-demand-response">Open ADR 3.0</a>
```
Comment (erikjee) [inline templates/components/nav.html:25]:
```suggestion
          <a class="subitem" href="/initiatives/automated-demand-response">Automated Demand Reponse</a>
```

**Review by erikjee — COMMENTED** [review]
Looks good. I'll leave tweaking the text to @LisetRoos.

File: content/initiatives/automated-demand-response.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+This project is building a modern, open-source Rust implementation of OpenADR 3.0, the latest version of OpenADR, the automated demand response standard that's being developed by the [OpenADR Alliance](https://www.openadr.org/).
+
+Written in [Rust](https://www.rust-lang.org/), our implementation is fast, lightweight, reliable, and can be used in many different environments. It will be an important building block for a multitude of digital products and services required for the further electrification of our energy systems and the energy transition.
+
+### What We've Done
+
+[Tweede golf](https://tweedegolf.nl/en) started development in June 2024 with initial funding from [ElaadNL](https://elaad.nl/en/), the Knowledge & Innovation center for the joint Dutch grid operators that researches and tests smart and sustainable charging of electric vehicles.
+
+To enable global collaboration and ensure neutral governance, in October 2024 the project became part of the [Linux Energy Foundation OpenLEADR](https://lfenergy.org/projects/openleadr/) project, that was already hosting an OpenADR 2.0b implementation written in Python.
```
Comment (LisetRoos) [inline content/initiatives/automated-demand-response.md:28]:
```suggestion
To enable global collaboration and ensure neutral governance, the project became part of the [Linux Energy Foundation OpenLEADR](https://lfenergy.org/projects/openleadr/) project  in October 2024. OpenLEADR were already hosting an OpenADR 2.0b implementation written in Python.
```

File: content/initiatives/automated-demand-response.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+The reponsitory for the OpenADR 3.0 Rust implementation is called [openleadr-rs](https://github.com/OpenLEADR/openleadr-rs).
+
+### Current Status
+
+Currently (November 2024) the project is seeing its first external contributions and is almost ready to do a first field-test-ready release, for which we're seeking funding.
```
Comment (LisetRoos) [inline content/initiatives/automated-demand-response.md:34]:
```suggestion
Currently (November 2024) the project is seeing its first external contributions and is almost ready for a field-test-ready release, for which we're seeking funding.
```

File: content/initiatives/automated-demand-response.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+### What's Next
+
+After inital funding by ElaadNL and Tweede golf in 2024, the project is currently (November 2204) **seeking a total of EUR 40.000 in funding** for:
+
+**In Dec 2024 / Jan 2025:**
++ Doing a first field-test-ready release
```
Comment (LisetRoos) [inline content/initiatives/automated-demand-response.md:41]:
```suggestion
+ The first field-test-ready release 
```

File: content/initiatives/automated-demand-response.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+After inital funding by ElaadNL and Tweede golf in 2024, the project is currently (November 2204) **seeking a total of EUR 40.000 in funding** for:
+
+**In Dec 2024 / Jan 2025:**
++ Doing a first field-test-ready release 
+
+**During 2025:**
++ Ongoing development and maintenance
++ Continued community building and contributor support
++ Additional releases when new (minor) versions of the OpenADR starndard are published
++ Preparing for a first long term support release
```
Comment (LisetRoos) [inline content/initiatives/automated-demand-response.md:47]:
```suggestion
+ Preparing for a long-term support release
```

File: content/initiatives/automated-demand-response.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+If you are interested in funding this work, please get in touch with us via <a href="mailto:donate@trifectatech.org">donate@trifectatech.org</a> and request the project's sponsorship prospectus.
```
Comment (LisetRoos) [inline content/initiatives/automated-demand-response.md:49]: What if we make this a direct download? (This is a suggestion for all projects that we have a prospectus for.)

**Review by LisetRoos — CHANGES_REQUESTED** [review]

File: content/initiatives/automated-demand-response.md (hunk 8 above)
Comment (erikjee) [reply content/initiatives/automated-demand-response.md:55]: Maybe in the future, but for now we keep the prospectus private; for prospects only

**Pull request merged**